python/artm/regularizers.py

```diff
@@ -66,29 +67,63 @@ def _update_config(self, obj, first=False):
 
 
 class Regularizers(object):
-    def __init__(self, master):
+    def __init__(self, master, force_add=False):
```

Comment:
Instead of `force_add` I would rather create two methods: `add` and `add_or_replace` (first one is strict, second allows to add even it regularizer exists).